client/GameBoard.jsx

```diff
@@ -286,49 +286,80 @@ export class InnerGameBoard extends React.Component {
         return (
             <div className='game-board'>
                 <div className='main-window'>
-                    <PlayerRow agenda={otherPlayer ? otherPlayer.agenda : undefined}
-                        faction={otherPlayer ? otherPlayer.faction : undefined}
-                        hand={otherPlayer ? otherPlayer.hand : []} isMe={false}
-                        numDrawCards={otherPlayer ? otherPlayer.numDrawCards : 0}
-                        discardPile={otherPlayer ? otherPlayer.discardPile : []}
-                        deadPile={otherPlayer ? otherPlayer.deadPile : []}
-                        onCardClick={this.onCardClick}
-                        onMouseOver={this.onMouseOver}
-                        onMouseOut={this.onMouseOut}
-                        />
-                    <div className='middle'>
-                        <div className='left-side'>
+                    <div className='left-side'>
+                        <div className='player-info'>
                             <PlayerStats gold={otherPlayer ? otherPlayer.gold : 0} claim={otherPlayer ? otherPlayer.claim : 0}
                                 reserve={otherPlayer ? otherPlayer.reserve : 0} power={otherPlayer ? otherPlayer.totalPower : 0} user={otherPlayer ? otherPlayer.user : null} />
-                            <div className='plots-pane'>
+                            <div className='deck-info'>
+                                <div className='deck-type'>
+                                    <CardCollection className='faction' source='faction' cards={[]} topCard={otherPlayer ? otherPlayer.faction : undefined} onMouseOver={this.props.onMouseOver} onMouseOut={this.props.onMouseOut} disablePopup />
+                                    {otherPlayer && otherPlayer.agenda && otherPlayer.agenda.code !== '' ?
+                                        <CardCollection className='agenda' source='agenda' cards={[]} topCard={otherPlayer ? otherPlayer.agenda : undefined} onMouseOver={this.props.onMouseOver} onMouseOut={this.props.onMouseOut}
+                                              onClick={this.onClick} disablePopup onMenuItemClick={this.props.onMenuItemClick} />
```

Comment:
As with the other spot, `this.onMenuItemClick` instead of `this.props.onMenuItemClick`.